<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../core/core.qrc">
    <normaloff>:/resources/icon64.png</normaloff>:/resources/icon64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ComputerMonitoringView" name="computerMonitoringView" native="true">
      <widget class="QToolButton" name="autoFitButton">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>570</y>
         <width>69</width>
         <height>37</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Adjust optimal size</string>
       </property>
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/zoom-fit-best.png</normaloff>:/resources/zoom-fit-best.png</iconset>
       </property>
      </widget>
      <widget class="QSlider" name="gridSizeSlider">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>580</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="computerManagementButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>570</y>
         <width>227</width>
         <height>37</height>
        </rect>
       </property>
       <property name="text">
        <string>Computer rooms</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/applications-education.png</normaloff>:/resources/applications-education.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QToolButton" name="screenshotManagementButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>570</y>
         <width>158</width>
         <height>37</height>
        </rect>
       </property>
       <property name="text">
        <string>Screenshots</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/camera-photo.png</normaloff>:/resources/camera-photo.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QLabel" name="spacerLabel1">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>570</y>
         <width>31</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="aboutButton">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>570</y>
         <width>71</width>
         <height>37</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>About Veyon</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset resource="../../core/core.qrc">
         <normaloff>:/resources/help-about.png</normaloff>:/resources/help-about.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="spacerLabel3">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>570</y>
         <width>99</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="filterLineEdit">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>570</y>
         <width>113</width>
         <height>38</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Search users and computers</string>
       </property>
      </widget>
      <widget class="QLabel" name="spacerLabel2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>570</y>
         <width>31</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="alignComputersButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>570</y>
         <width>69</width>
         <height>37</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Align computers to grid</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/align-grid.png</normaloff>:/resources/align-grid.png</iconset>
       </property>
      </widget>
      <widget class="QToolButton" name="useCustomComputerPlacementButton">
       <property name="geometry">
        <rect>
         <x>960</x>
         <y>570</y>
         <width>69</width>
         <height>37</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Use custom computer placement</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/exchange-positions-zorder.png</normaloff>:/resources/exchange-positions-zorder.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="spacerLabel4">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>570</y>
         <width>99</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="filterPoweredOnComputersButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>570</y>
         <width>69</width>
         <height>37</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Only show powered on computers</string>
       </property>
       <property name="icon">
        <iconset resource="../master.qrc">
         <normaloff>:/resources/powered-on.png</normaloff>:/resources/powered-on.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MainToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainToolBar</class>
   <extends>QToolBar</extends>
   <header>MainToolBar.h</header>
  </customwidget>
  <customwidget>
   <class>ComputerMonitoringView</class>
   <extends>QWidget</extends>
   <header>ComputerMonitoringView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../core/core.qrc"/>
  <include location="../master.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>968</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomComputerPlacementButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>alignComputersButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>1134</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAboutDialog()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
